test_diff_drive_controller:
  ros__parameters:
    base_frame_id: base_link

    left_wheel_names: ['left_wheel_joint']
    right_wheel_names: ['right_wheel_joint']
    wheel_separation: 0.1475
    wheel_radius: 0.033